### PR TITLE
FEAT: Add lint preventing EIP references in backticks

### LINKED ISSUE #90: Add lint preventing EIP references in backticks
Many authors attempt to circumvent the EIP link requirements by putting EIP-XXXX in backticks (eg. \`EIP-1234\`.) This is bad form, and should be rejected.

### PR DESCRIPTION
**Pull request by VictoriaAde**
Fixes #90

### CODE REVIEW

File: eipw-lint/src/lints/markdown/no_backticks.rs [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+ {
+     fn lint<'a>(&self, slug: &'a str, ctx: &Context<'a, '_>) -> Result<(), Error> {
+         let pattern = self.0.as_ref();
+         let re = Regex::new(pattern).map_err(Error::custom)?;
+         let mut visitor = Visitor {
+             ctx,
+             re,
+             pattern,
+             slug,
+             found_backticks: false,
```
Comment (SamWilsn) [inline eipw-lint/src/lints/markdown/no_backticks.rs:37]: I don't see where this member is used?

Comment (VictoriaAde) [reply eipw-lint/src/lints/markdown/no_backticks.rs:37]: Hi, I'm sorry for the late response.
What member are you referring to?

Comment (SamWilsn) [reply eipw-lint/src/lints/markdown/no_backticks.rs:37]: `found_backticks`

Comment (SamWilsn) [reply eipw-lint/src/lints/markdown/no_backticks.rs:37]: It seems like it gets set to true, but its value is never read.

Comment (VictoriaAde) [reply eipw-lint/src/lints/markdown/no_backticks.rs:37]: Oh, I just noticed, I will fix it.

**Comment by VictoriaAde** [comment]
Hello @SamWilsn, I apologies for the delay in closing this issue. 
I have some updates on this...

Given that the implementation looks correct, the issue might be how the AST is being constructed or traversed. I’m currently investigating how the markdown is being parsed into an AST. 

Please, let me know if you have any thoughts on this.

**Pull request merged**